media-fonts/jetbrains-mono/jetbrains-mono-1.0.4.ebuild

```
@@ -0,0 +1,20 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit font
+
+DESCRIPTION="A free and open-source typeface for developers"
+HOMEPAGE="https://www.jetbrains.com/lp/mono/"
+SRC_URI="https://github.com/JetBrains/JetBrainsMono/releases/download/v${PV}/JetBrainsMono-${PV}.zip -> ${P}.zip"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+BDEPEND="app-arch/unzip"
+
+S="${WORKDIR}/JetBrainsMono-${PV}"
+FONT_S="${S}/ttf"
```

```suggestion
S="${WORKDIR}/JetBrainsMono-${PV}/ttf"
```
There are no relevant files outside of the `ttf` dir, so `S` could just be that subdir.

---

Applied the change.